### DISCUSSION TITLE
control of label fill color

### DISCUSSION THREAD

**Discussion by ghs101** [jokergoo/circlize#341] [Q&A] [answered]
is it possible to include label parameters to control fill of label  independently from label text color. like in the example 

![fmicb-10-00633-g001](https://user-images.githubusercontent.com/79457449/194691518-a2b194c6-f9f8-4bd4-8c48-4125c1db5dc1.jpg)

**Comment by jokergoo** [comment, marked as answer]
There is a package called **gridtext** which can do this, but it is built with the **grid** graphics system. **circlize** is built with the  base graphics, so currently it does not support. Maybe in the future when I have time, I can try to implement it.
